Fluid/Values/DictionaryDictionaryFluidIndexable.cs

```diff
@@ -22,7 +22,11 @@ public IEnumerable<string> Keys
             {
                 foreach (var key in _dictionary.Keys)
                 {
-                    yield return key.ToString();
+                    // Only handle string keys since this is what TryGetValue returns
```

Comment:
This class has odd `TryGetValue`, it probably should not do`Contains` + indexing, but unrelated to this PR.

Reply:
Not worried about this one. It's just for cases where they don't implement anything more interesting than `IDictionary`.